quinn/src/platform/fallback.rs

```diff
@@ -30,7 +32,13 @@ impl super::UdpExt for UdpSocket {
         Ok(sent)
     }
 
-    fn recv_ext(&self, buf: &mut [u8]) -> io::Result<(usize, SocketAddr, Option<EcnCodepoint>)> {
-        self.recv_from(buf).map(|(x, y)| (x, y, None))
+    fn recv_ext(&self, bufs: &mut [IoSliceMut<'_>], meta: &mut [RecvMeta]) -> io::Result<usize> {
+        let (len, addr) = self.recv_from(&mut bufs[0])?;
```

Comment:
We'll eventually be using a more efficient syscall on windows, of course. Do you have an idea for how this might be improved (for macOS as well) in the mean time?

Reply:
A simple improvement is setting `BATCH_SIZE` dependent on the platform, and to `1` on windows

Reply:
Good call; done.